Gemfile

```diff
@@ -58,8 +58,10 @@ group :development, :test do
   gem "launchy"
   gem "orderly"
   gem "simplecov"
-  gem "faker"
-  gem "factory_bot_rails"
+  gem 'factory_bot_rails'
+ gem 'faker'
+ gem 'rails-controller-testing'
```

Comment:
This worked this time.  I'm not sure why I wasn't able to install it before, curious.